registry/templates/create_bucket

```diff
@@ -5,13 +5,22 @@ import sys
 from boto.s3.connection import OrdinaryCallingFormat
 
 conn = boto.connect_s3(
+    {{ if exists "/deis/store/gateway/accessKey" }}
     aws_access_key_id='{{ getv "/deis/store/gateway/accessKey" }}',
     aws_secret_access_key='{{ getv "/deis/store/gateway/secretKey" }}',
+    {{end}}
+    {{ if exists "/deis/store/gateway/host" }}
     host='{{ getv "/deis/store/gateway/host" }}',
     port={{ getv "/deis/store/gateway/port" }},
+    {{ end }}
     is_secure=False,
     calling_format=OrdinaryCallingFormat())
+
+{{ if exists "/deis/registry/s3bucket" }}
+name = '{{ getv "/deis/registry/s3bucket" }}'
```

Comment:
I think this is what was confusing me:

https://github.com/laurrentt/deis/blob/registry-s3-assume-role/registry/templates/config.yml#L83
https://github.com/laurrentt/deis/blob/registry-s3-assume-role/registry/templates/config.yml#L102

I'm fine with this since it's old behaviour, but it does look like we're using 2 different keys for (generally) the same thing.